docs/advanced/private-registries/gcr.md

```diff
@@ -1,7 +1,41 @@
-Trivy uses Google Cloud SDK. You don't need to install `gcloud` command.
+# Requirements
+None, Trivy uses Google Cloud SDK. You don't need to install `gcloud` command.
 
-If you want to use target project's repository, you can settle via `GOOGLE_APPLICATION_CREDENTIAL`.
+# Privileges
+Credential file should have the `roles/storage.objectViewer` permissions.
```

Comment:
I suppose the permission is necessary according to the document. Please correct me if I'm wrong.

```suggestion
Credential file must have the `roles/storage.objectViewer` permissions.
```